nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/files/impl/FileStorageManager.java

```diff
@@ -36,12 +41,14 @@ public void saveFile(String username, String uploadId, String documentId,
         try {
             stream = new BufferedOutputStream(new FileOutputStream(file));
         } catch (FileNotFoundException e) {
+            systemMessageHandler.handleError("Could not store file.", e);
```

Comment:
exceptions need to be either handled or thrown. It's thrown here, so don't handle it (or there will be duplicate entries in September for the same exception).